examples/1_Simple/tracing_fieldline.py

```diff
@@ -86,16 +86,45 @@ def trace_fieldlines(bfield, label):
 # trace_fieldlines(bs, 'bs')
 
 
-n = 16
+# Bounds for the interpolated magnetic field chosen so that the surface is
+# entirely contained in it
+n = 20
 rs = np.linalg.norm(s.gamma()[:, :, 0:2], axis=2)
 zs = s.gamma()[:, :, 2]
 rrange = (np.min(rs), np.max(rs), n)
 phirange = (0, 2*np.pi/3, n*2)
 zrange = (0, np.max(zs), n//2)
```

Comment:
Shouldn't the interpolant cover negative z values too?

Reply:
The same comment applies also to the analogous line of `tracing_particles.py`.

Reply:
no, we're exploiting stellarator symmetry in the interpolant, so negative `z` values are mapped to `-z`